src/main.cpp

```diff
@@ -69,6 +69,22 @@ int main(const int argc, const char *const *const argv) {
         ldLog() << LD_ERROR << "No such directory:" << appDirPath.Get() << std::endl;
         return 1;
     }
+    
+    int qtMajorVersion = 5;
```

Comment:
Why not just ask `qmake`? Call it with `-v`, it shows you the Qt version in use. It's easier than having the user specify this.

Reply:
I will check, qmake -v output needs to be parsed to extract the major version.

Example qmake -v output:

```
➜  ~ qmake -v
QMake version 2.01a
Using Qt version 4.8.7 in /usr/lib/x86_64-linux-gnu

```

Reply:
I will use `qmake -query QT_VERSION` it is perfect for the use case:

```
➜  ~ qmake -query QT_VERSION
4.8.7
➜  ~ Qt/5.15.0/gcc_64/bin/qmake -query QT_VERSION
5.15.0
➜  ~ qt-6.0.0/bin/qmake -query QT_VERSION
6.0.0
```

Reply:
QT_VERSION is already available in `qmakeVars` so it will use that.

Reply:
Oh right... I should've remembered that, I guess. Well, it makes it a lot simpler, I guess.

Reply:
it worked like a charm